perl-lib/OESS/lib/OESS/NSI/Provisioning.pm

```diff
@@ -32,7 +32,7 @@ use Data::Dumper;
 use GRNOC::Log;
 use GRNOC::Config;
 use GRNOC::WebService::Client;
-
+use JSON::XS;
```

Comment:
Not sure if this a new dependency or not. If so we've been using the following elsewhere:
```
use JSON;

my $json = encode_json($obj);
my $obj  = decode_json($json);